File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {

        int numsLen = nums.length;

        int[] newNums = new int[numsLen * 2];

        for(int i = 0; i < numsLen; i++){
            newNums[i] = nums[i];
            newNums[i + numsLen] = nums[i];
        }

        return newNums;
        
    }
}

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number[]}
     */
    productExceptSelf(nums) {
        
        let ans= []
        for(let i=0; i<nums.length;i++){
           let val = 1
           for(let j= 0 ;j< nums.length;j++){
            if(i != j){
                val *= nums[j]
            }
           }
           ans.push(val)
        }
        return ans
    }
}
